### PR TITLE
Add required python-six package to installation

### LINKED ISSUE #3020: CentOS 7.3 image on GCE fails on "ImportError: No module named six"
Running the [OpenShift Origin](https://docs.openshift.org/latest/install_config/install/advanced_install.html#running-the-advanced-installation) install on GCE with CentOS 7.3 (centos-7-v20161212) images fails on `Gather Cluster facts` task with:

```
...
TASK [openshift_facts : Gather Cluster facts and set is_containerized if needed] ***
fatal: [104.199.88.43]: FAILED! => {"changed": false, "failed": true, "module_stderr": "Shared connection to 104.199.88.43 closed.\r\n", "module_stdout": "/etc/profile.d/lang.sh: line 19: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory\r\nTraceback (most recent call last):\r\n  File \"/tmp/ansible_wO3uEu/ansible_module_openshift_facts.py\", line 29, in <module>\r\n    from six import string_types\r\nImportError: No module named six\r\n", "msg": "MODULE FAILURE"}
...
```

Specifically, `ImportError: No module named six`.

Using this [SO answer](http://stackoverflow.com/a/13967865/2408961) and installing `six` module with `easy_install six` on all hosts and rerunning the playbook, results in a successful install.

##### Version
`git describe`
openshift-ansible-3.4.17-1-283-g930102c3

`ansible --version`
ansible 2.2.0.0

##### Steps To Reproduce
1. Provision hosts on GCE using CentOS 7.3 (centos-7-v20161212) image - see [this terraform plan](https://gist.github.com/donovanmuller/23f902ffd637ce9f2022e6f4d7cfe756)
2. Run the `byo` playbook, `ansible-playbook -i ../gce/gce playbooks/byo/config.yml`

##### Current Result

`openshift_facts` task fails with `ImportError: No module named six`.
Need to manually install `six` module with `easy_install six` and rerun playbook.

##### Expected Result

Successful installation

##### Additional Information
CentOS 7.3 (centos-7-v20161212) on GCE

### PR DESCRIPTION
**Pull request by tbielawa**
* Moves package list into a defaults/main.yml file
* Closes #3020

### CODE REVIEW

**Comment by tbielawa** [comment]
aos-ci-test

**Comment by tbielawa** [comment]
@sdodson i can't recall, are we supposed to use a special prefix for that variable I created?

**Comment by sdodson** [comment]
> @sdodson i can't recall, are we supposed to use a special prefix for that variable I created?

no, but use vars/main.yml instead since it's 'private' rather than 'public' ?

File: roles/openshift_facts/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+required_packages:
+  - PyYAML
+  - yum-utils
+  - python-six
```
Comment (detiber) [inline roles/openshift_facts/defaults/main.yml]: python-dbus and iproute should also probably be included in this list.

**Comment by tbielawa** [comment]
@sdodson wrote:

> no, but use vars/main.yml instead since it's 'private' rather than 'public' ?

done diddely do done

@detiber wrote:

> python-dbus and iproute should also probably be included in this list.

diddley doodley done

**Comment by tbielawa** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Pull request merged**